scionlab/models/core.py

```diff
@@ -1210,6 +1210,10 @@ class Service(models.Model):
         (BW, 'Bandwidth tester server'),
         (PP, 'Pingpong server'),
     )
+    STATIC_SERVICE_TYPES = (
```

Comment:
Not too sure on the name "static" -- without knowing about the special case for systemd, these services are no more _static_ than any of the others.
Maybe "extra services" (that's what they're called in other places)?